src/components/Table/index.tsx

```diff
@@ -1,2 +1,80 @@
-export * from './Table'
-export * from './interfaces'
+import React from 'react'
+
+type T = { [key: string]: any }
+
+interface Props {
+  tableClassName?: string
+  headerClassName?: string
+  columns: { key: string; label: string; accessor?: (row: T) => React.ReactNode }[]
```

Comment:
change access to `formatter` where `formatter` is `(row: T) => string | React.ReactNode`

Reply:
i feel, with the key in each column, things will be easier than without it. for example, when adding a filter, we'd want to know that column's key, and then loop through the data and look at that key's value, etc.?